scripts/script.js

```diff
@@ -0,0 +1,163 @@
+const newTitle = document.getElementById('new-title'); // eslint-disable-line max-classes-per-file
```

Comment:
- [ ] You have **disabled ESLint checks** on parts of your code. Please enable **all** ESLint checks on your code and try modifying that portion of your code to adhere to ESLint standards. This will ensure a robust and clean code.

Reply:
@fdezcaminero That is correct! Kindly submit your project at once and it will be approved! Cheers! 😃

Reply:
@rloterh Thank you so much, friend!